src/Command/SymlinksCommand.php

```diff
@@ -94,7 +94,7 @@ public function generateSymlinks($rootDir, OutputInterface $output)
      */
     private function symlinkFiles($uploadPath, $rootDir, OutputInterface $output)
     {
-        $finder = $this->findIn('.public', "$rootDir/$uploadPath");
+        $finder = $this->findIn("$rootDir/$uploadPath")->files()->name('.public');
```

Comment:
What's the reason for filtering the `files` directory for `.public` files? Shouldn't the `files` directory be public anyway?